Feature page

emmanuellekamwa opened the pull request:
The aim of this project was to clearly understand the concept of background-images and gradients, and how to manipulate them properly and efficiently. In this project, the concept of background-images and gradients are respected and well used. To design this page I have made use of:
- Flex
- Relative Positioning
 I hope to have a good feedback from you. Feel free to comment.

emmanuellekamwa committed:
homepage: setting up the html part of the page

emmanuellekamwa committed:
homepage: styling the page and correcting some errors

emmanuellekamwa committed:
homepage: adding stickler file and styling file to the page

emmanuellekamwa committed:
homepage: adding a gitignore file to the repository

emmanuellekamwa committed:
homepage: fixing css errors, modifying the nav-bar and resizing the page

emmanuellekamwa committed:
homepage: adjusting the position of the links inside the nav-bar

emmanuellekamwa committed:
homepage: readjusting the margin-top between the background-image and the list of images

emmanuellekamwa committed:
homepage: readjusting the margin-top between the background-image and the list of images

emmanuellekamwa committed:
homepage: adjusting the position of the nav-bar and the position of the titles

emmanuellekamwa committed:
homepage: fixing the linear-gradient of the nav-bar

emmanuellekamwa committed:
homepage: updating my README file

trekab reviewed (changes requested):
# Changes Required!
Hi @emmanuellekamwa, you have done an impressive job with the navigation gradient! :clap: However, there are a few changes you need to make to this project. These include;

### Mandatory:
- [ ] The `Live demo` link in the README file doesn't link to the latest version of your work.
- [ ] Fix all linter errors.([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) form [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
- [ ] Your clone appears to be broken([Screenshot](https://github.com/trekab/images/blob/master/emma_clone.png?raw=true) from my screen). Please position the bottom elements like they are positioned in the [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) site.

### Optional:
- [ ] Your use of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) could be improved. Check out how to best use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp).
- [ ] Use the [CSS validator](https://jigsaw.w3.org/css-validator/) to catch CSS errors.

__Kindly make the necessary changes and submit another code review request__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Treasure Kabareebe__

emmanuellekamwa committed:
homepage: fixing css issues

emmanuellekamwa committed:
Update style.css

indented line 7

emmanuellekamwa committed:
Update style.css

indented line 3

emmanuellekamwa committed:
Update style.css

indented from line 11 to line 25

emmanuellekamwa committed:
Update style.css

indented line 48, 71 and 89

emmanuellekamwa committed:
Update style.css

indented line 47

emmanuellekamwa committed:
Update style.css

indented line 71

emmanuellekamwa committed:
Update style.css

indented from line 29 to line 198

emmanuellekamwa committed:
Update style.css

Indented from line 90 to line 94

emmanuellekamwa committed:
homepage: changing the value of the font-weight from 3em to 10px

emmanuellekamwa committed:
homepage: fixing conflicts in style.css

emmanuellekamwa committed:
homepage: removing the font-weight property

emmanuellekamwa committed:
homepage: changing the demo link of the README file

emmanuellekamwa commented:
I have made some changes on my code as suggested by the TSE as correcting the CSS code and the HTML semantics. I have also updated the README file and I have corrected the linters. Hope to get positive reviews

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

addod19 reviewed (changes requested):
## Status: Require changes:recycle:  :traffic_light: 

Hi @emmanuellekamwa, you did a great job fixing all the comments by the previous TSE :smile: However, I found 2 situations I would like you to fix before proceeding. Kindly address the comments below:
- Kindly fix the linter errors, almost all the errors are about indentations. Easy fix
- Kindly address and fix the image below. Some of the images are not showing and the ones showing are also overlapping with the main content
![now](https://user-images.githubusercontent.com/46388113/82492696-23a64700-9ad6-11ea-935b-ef118d590120.png)


Kindly fix these and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

emmanuellekamwa committed:
Update style.css

indented from line 3 to line 49

emmanuellekamwa committed:
Update style.css

indented from line 50 to line 94

emmanuellekamwa committed:
homepage: changing the margin-top of the images and fixing linter errors

emmanuellekamwa commented:
I have corrected other errors mentioned by the TSE. I hope to have a positive review this time

lucasmazo32 commented on index.html:
```diff
@@ -0,0 +1,136 @@
+                </header>
+            </main>
+            </div>
+            <div class="flex-container">
+                <div class="flex-item-div">
+                    <a href="#"
+                        ><img src="images/iPhone5s.jpg" alt="phone"
+                    /></a>
+                </div>
+                <div class="flex-item-div">
+                    <a href="#"
+                        ><img src="images/iPhone5c.jpg" alt="phone"
+                    /></a>
+                </div>
```
- [ ] These two particular images are not being displayed. I noticed that is about a type. The files are saved as iphone5s and iphone5c, not iPhone5s and iPhone5c. Kindly fix that.

> screenshot 
![image](https://user-images.githubusercontent.com/57414789/82617074-8f4ee980-9b94-11ea-9bf1-43998412c393.png)

lucasmazo32 commented on index.html:
```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
```
- [ ] I get some W3C validator. Kindly check that.

> screenshot
![image](https://user-images.githubusercontent.com/57414789/82617109-b73e4d00-9b94-11ea-813f-33ab284f654f.png)

lucasmazo32 reviewed (changes requested):
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

emmanuellekamwa committed:
homepage: changing the file names of my images and resolving html validator issues

emmanuellekamwa commented:
I have resolved another set of errors made mention of by the TSE and I think that now my project is going to be approved.

sumancrest0001 reviewed (commented):
**Status: changes required
Hello @emmanuellekamwa,**
You have done a good job so far and fixed all the issues raised by the previous TSE. But we need to make few minor changes before moving ahead. Please fix them and request another code review.

[NOTE] optional changes are not mandatory but nice to have.

sumancrest0001 commented on style.css:
```diff
@@ -0,0 +1,198 @@
+  list-style-type: none;
+}
+
+.item a {
+  text-decoration: none;
+  color: white;
+}
+
+.titles a {
+  color: white;
+  text-decoration: none;
+}
```
Let's remove the repeated CSS stylings.

sumancrest0001 commented on style.css:
```diff
@@ -0,0 +1,198 @@
+}
+
+/* bottom page */
+.footer-flex {
+  max-width: 82%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  font-family: "Cabin", sans-serif;
+  font-size: 12px;
+  color: #666;
+}
+
+.copyright-flex {
+  max-width: 82%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  font-family: "Cabin", sans-serif;
+  font-size: 12px;
+  color: #666;
+}
+
+.footer-links {
+  max-width: 82%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
```
Let's remove the repeated stylings and follow the DRY principle.

sumancrest0001 reviewed (commented):
[Optional] We can change the font-size, font-family, and font-weight of the texts on the main section to give it a similar look to the original Apple page.
Currently :point_down: 
![Screenshot from 2020-05-23 08-28-49](https://user-images.githubusercontent.com/37747619/82720028-84e53a80-9ccf-11ea-880d-5b53e73218f9.png)

It should be :point_down: 
![Screenshot from 2020-05-23 08-29-00](https://user-images.githubusercontent.com/37747619/82720033-8f9fcf80-9ccf-11ea-8ebf-8e8e071c586e.png)

sumancrest0001 reviewed (changes requested):
**Kindly make the requested changes and submit the project another code review.**

Need help or have questions regarding the code review? DM me on slack **Suman Shrestha**
**Happy coding !!!**

emmanuellekamwa committed:
homepage: removing repeated styling in my css and changing the font-size and font-family of the page

emmanuellekamwa committed:
homepage: correcting linter errors

emmanuellekamwa commented:
Corrections have been made again as pointed by the TSE. I am eager to hear from you.

sumancrest0001 commented on index.html:
```diff
@@ -0,0 +1,134 @@
+                </header>
+            </main>
+        </div>
+        <div class="flex-container">
+            <div class="flex-item-div">
+                <a href="#"><img src="images/iphone5s.jpg" alt="phone" /></a>
+            </div>
+            <div class="flex-item-div">
+                <a href="#"><img src="images/iphone5c.jpg" alt="phone" /></a>
+            </div>
+            <div class="flex-item-div">
+                <a href="#"><img src="images/Mac.jpg" alt="phone" /></a>
+            </div>
+            <div class="flex-item-div">
+                <a href="#"><img src="images/supplier.jpg" alt="phone" /></a>
+            </div>
+        </div>
```
**Status: required changes
Hello @emmanuellekamwa** 
Thank you for addressing the raised issues but it seems that some issues are introduced while making the changes. Please fix the following minor issues to get approval for this project. 
1. The four-articles section is overlapping with the main section. one of the images is not visible in [githack](https://raw.githack.com/)
2. left and right footer sections are overlapping. There is no clear gap between them.
Please consider the following image :point_down: 
![Screenshot from 2020-05-25 09-33-02](https://user-images.githubusercontent.com/37747619/82778913-7daa6200-9e72-11ea-9b06-f0dd72914b99.png)

sumancrest0001 reviewed (changes requested):
**Please make these changes and request another review.
need help or have a query? Contact me on slack Suman Shrestha
Happy coding**

emmanuellekamwa committed:
homepage: correcting linter issues, changing the margin-top of the images and the gap between the footers

emmanuellekamwa commented:
Changes have been made in my project.

santiago-dev96 commented:
# Pull Request under review by @santiago-rodrig (TSE)

santiago-dev96 reviewed (approved):
# Project approved! :100:
You still have a lot to learn, I saw several uses of negative margins and other not so good practices; still, you fulfill the requirements for approval and it seems that you've been trying to get this project approved several times already. You may continue with the next project, but I suggest you to [take a read to this link](https://developer.mozilla.org/en-US/docs/Learn/CSS), pay extra attention to CSS building blocks and CSS layout sections.

Good luck on your next projects! :wave:
